src/lib/Microsoft.Fx.Portability.Cci/Analyzer/DependencyFinderEngine.cs

```diff
@@ -99,16 +107,15 @@ private IEnumerable<MemberDependency> GetDependencies(string assemblyLocation)
                     {
                     }
 
-                    HashSet<string> newValue = new HashSet<string>
-                    {
-                        callingAssembly
-                    };
+                    var newValue = new HashSet<string> { callingAssembly };
+
                     _unresolvedAssemblies.AddOrUpdate(e.Unresolved.Format(), newValue, (key, existingHashSet) =>
                     {
                         lock (existingHashSet)
```

Comment:
More unneeded locks. I guess maybe don't mess with this too much. We can make another pass at some point to simplify.

Reply:
I would prefer to take care of these in another pass.  Open https://github.com/microsoft/dotnet-apiport/issues/810 to track the issue.